frontend/graphql.schema.json

```diff
@@ -0,0 +1,1539 @@
+{
```

Comment:
Afaik, artemis can also handle `.graphql` schemas (the dsl format, not in json). Personally, I'd rather like to have the .graphql files here. I will write a generator for these quickly...
I think the ones I can generate are not >1500 lines :)